sql/schema.sql

```diff
@@ -549,6 +549,15 @@ create table pranger (
 create index pranger_at_idx
           on pranger (at desc);
 
+-- Alliance members
```

Comment:
The comment probably should say: `User password reset tokens`.

Reply:
🤦‍♂️